site/en/docs/extensions/mv3/storage-and-cookies/index.md

```diff
@@ -0,0 +1,116 @@
+This applies to any data stored using web storage APIs (such as
+[Local Storage][local-storage] and [IndexedDB][indexeddb]).
+
+By default, extensions are subject to the normal quota restrictions on storage, which can be checked
+by calling `navigator.storage.estimate()`. Storage can also be evicted under heavy memory
+pressure, although this is rare. To avoid this:
+
+- Request the `"unlimitedStorage"` permission, which affects both extension and web storage APIs and
+exempts extensions from both quota restrictions and eviction.
+- Call [`navigator.storage.persist()`][storage-persist] for protection against eviction.
+
+Extension storage is shared across the extension's origin including the extension service worker,
+any extension pages (including popups and the side panel), and offscreen documents. In content
+scripts, calling web storage APIs accesses data from the host page the content script is injected on
+and not the extension.
+
+### Access in service workers {: #storage-in-service-workers }
+
+Some web storage APIs like [IndexedDB][indexeddb] are accessible in service workers. However,
+[Local Storage][local-storage] and [Session Storage][session-storage] are not.
+
+If you have a specific use case, use an [offscreen document][offscreen] to access these.
+
+### Partitioning {: #storage-partitioning }
+
+Partitioning is where keys are introduced for stored data to limit where it can be accessed.
+
+When enabled, [storage partitioning][storage-partitioning] introduces changes to how partitioning
+keys are defined to prevent certain types of cross-site tracking. In practice, this means that if
+site A embeds an iframe containing site B, site B will not be able to access the same storage it
+would usually have when navigated to directly.
+
+To mitigate the impact of this in extensions, two key exemptions apply:
+
+- If a page with the `chrome-extension://` scheme is embedded in any site, storage partitioning will
+not apply, and the extension will have access to its top-level partition.
+- If a page with the `chrome-extension://` scheme includes an iframe, and the extension has
+[host permissions][declare-permissions] for the site it is embedding, that site will also have
+access to its top-level partition.
+
+## Cookies {: #cookies }
+
+Cookies provide a way to store key/value pairs associated with a specific domain and path. They have
+limited value in extensions but understanding their behavior can be useful if you have a specific
+use case or have bundled a third-party script which uses them in its implementation.
+
+### Secure cookies {: #secure-cookies }
+
+The [`Secure`][cookies-restrict-access] cookie attribute is only supported for the `https://`
+scheme. Consequently, `chrome-extension://` pages are not able to set cookies with this attribute.
+
+This also means that extension pages cannot use other cookie attributes where the `Secure` attribute is
+required:
+
+- [`SameSite=None`][same-site]
+- [`Partitioned`][chips]
```

Comment:
What is this?

Reply:
An example of a cookie attribute that cannot be used in an extension. I linked to MDN here for anyone who wants to read more, without trying to document it a second time.

Reply:
Please add introductory text to explain that. The links should be in the introductory text.

Reply:
I'm not sure I understand. Isn't that covered by "extension pages cannot use other cookie attributes where the `Secure` attribute is required:"?

Reply:
I interpreted the word 'example' incorrectly. It looks fine.